proxy/contracts/mainnet/DepositBoxes/DepositBoxERC20.sol

```diff
@@ -267,7 +295,13 @@ contract DepositBoxERC20 is DepositBox, IDepositBoxERC20 {
         onlySchainOwner(schainName)
     {
         bytes32 schainHash = _schainHash(schainName);
-        _delayConfig[schainHash].bigTransferThreshold[token] = value;   
+        emit BigTransferThresholdIsChanged(
```

Comment:
Also would like to add tests to check are events emitted

Reply:
Updated.